aws-s3.sgnl.yaml

```diff
@@ -0,0 +1,91 @@
+description: "AWS S3 as a System of Record"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "S3-1.0.0"
+
+# Example Config:
+# To fetch entity data stored in a `data/internal` path in `employeeData`` S3 bucket in the `us-west-2` region i.e. s3://my-bucket/internal/{entity}.csv
+# {
+#   "region": "us-west-2",
+#   "bucket": "employeeData",
+#   "prefix": "data/internal",
+# }
+adapterConfig: "ewogICAgInJlZ2lvbiI6ICJ7e0lucHV0IFJlcXVpcmVkOiByZWdpb259fSIsCiAgICAiYnVja2V0IjogInt7SW5wdXQgUmVxdWlyZWQ6IGJ1Y2tldCBuYW1lfX0iCn0="
+
+# All auth mechanisms specified below must be supported by the specified Adapter.
+auth:
+  - basic:
+      username: "{{Input Required: AccessKeyID}}"
+      password: "{{Input Required: SecretAccessKey}}"
+
+# An example user entity
+entities:
+  User:
+    displayName: User
+    externalId: users # This must match the name of the CSV file in the S3 bucket
+    description: User represents the user data stored in AWS S3 in users.csv
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: userId
+        externalId: userId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: userName
+        externalId: userName
+        type: String
+
+  Group:
+    displayName: Group
+    externalId: groups # This must match the name of the CSV file in the S3 bucket
+    description: Group represents the group data stored in AWS S3 in groups.csv
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: groupId
+        externalId: groupId
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: groupName
+        externalId: groupName
+        type: String
+
+  GroupMember:
+    displayName: Group Member
+    externalId: members
+    description: Group Member represents the group membership data stored in AWS S3 in members.csv
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true # A unique ID is required for the entity
+      - name: userId
+        externalId: userId
+        type: String
+        indexed: true
+      - name: groupId
+        externalId: groupId
+        type: String
+        indexed: true
```

Comment:
Do User -> Group related relationships need to be created?

Reply:
Removing the example entities and relationships.